### ISSUE TITLE
Updating js-controller 5.x on Slave over admin "TypeError: Network Error when attempting to fetch ressource"

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.admin#1963]
**Describe the bug**  
I have just successfully updated my master to 5.0.5 via the Admin GUI.
When I tried to update my slave via the Admin GUI in the next step, I got the following error message:

Start update over admin UI
![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/d7ecb960-afb5-46c7-be4f-3199099356de)

Slave is successful updated
![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/d6d2c7fe-1009-4d32-a5fb-760bf0802db7)

First feedback from @foxriver76 "currently admin polls the wrong ip."

**Versions:**  
Master
Admin 6.5.9
js-controller 5.0.5

Slave
js-controller 5.0.4

**Screenshots & Logfiles**  
Log Slave

```
2023-06-23 20:24:12.448 - info: host.ioBrokerNew Controller will upgrade itself to version 5.0.5
2023-06-23 20:24:15.367 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Stopping controller
2023-06-23 20:24:15.463 - info: host.ioBrokerNew received SIGTERM
2023-06-23 20:24:15.474 - info: host.ioBrokerNew stopInstance system.adapter.history.1 (force=false, process=false)
2023-06-23 20:24:15.476 - info: host.ioBrokerNew stopInstance system.adapter.sql.0 (force=false, process=false)
2023-06-23 20:24:15.478 - info: host.ioBrokerNew stopInstance system.adapter.discovery.0 (force=false, process=false)
2023-06-23 20:24:15.514 - info: host.ioBrokerNew terminated
2023-06-23 20:24:19.599 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Successfully stopped js-controller
2023-06-23 20:24:19.618 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Server is running on http://localhost:8081
2023-06-23 20:24:55.444 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE]
changed 18 packages in 33s
81 packages are looking for funding
run `npm fund` for details
2023-06-23 20:24:56.129 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:24:57.143 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:24:58.148 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:24:59.149 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:00.160 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:01.169 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:02.174 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:03.184 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:04.189 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:06.204 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:07.209 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:08.221 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:09.225 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:10.236 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:11.240 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:12.247 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:13.253 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:14.257 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:15.267 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:16.273 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:17.280 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:18.280 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:19.285 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:20.296 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:21.298 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:22.309 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:23.308 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Final information delivered
2023-06-23 20:25:29.132 - info: host.ioBrokerNew [CONTROLLER_AUTO_UPGRADE] Successfully started js-controller
2023-06-23 20:25:31.159 - info: host.ioBrokerNew iobroker.js-controller version 5.0.5 js-controller starting
2023-06-23 20:25:31.170 - info: host.ioBrokerNew Copyright (c) 2014-2023 bluefox, 2014 hobbyquaker
2023-06-23 20:25:31.173 - info: host.ioBrokerNew hostname: ioBrokerNew, node: v18.16.0
2023-06-23 20:25:31.175 - info: host.ioBrokerNew ip addresses: 192.168.178.78 fe80::d4a0:37ff:fe8e:f0c1
2023-06-23 20:25:31.750 - info: host.ioBrokerNew connected to Objects and States
2023-06-23 20:25:31.796 - info: host.ioBrokerNew added notifications configuration of host
2023-06-23 20:25:32.976 - info: host.ioBrokerNew 61 instances found
2023-06-23 20:25:33.019 - warn: host.ioBrokerNew does not start any instances on this host
```

**Comment by foxriver76** [comment]
According to https://github.com/ioBroker/ioBroker.admin/commit/5dc9f88402fe32b4db3170c9abd869ad4cf7cef0 it should work with 6.5.9 but this seems to be the used version in this ticket.

**Comment by Feuer-sturm** [comment]
@foxriver76 According to the screenshot in the first post admin version was 6.5.9
I will retest this issue with current admin version v6.6.0 and the new js-controller 5.0.7

**Comment by foxriver76** [comment]
@Feuer-sturm Thanks a lot, let me know, however, probably it is still present unfortunately

**Comment by foxriver76** [comment]
When do so, please also check browser console for an error log like `Cannot find ip address: ....`

**Comment by Feuer-sturm** [comment]
@foxriver76 

Admin 6.6.0
update vom js-controller 5.0.6 auf 5.0.7

- Update ioBroker Master über Admin UI -> erfolgreich
- Update ioBroker Slave über Admin (welcher auf dem Master läuft) -> Fehlermeldung

*Screenshot*
![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/4fdd89b9-6f0a-4118-91ec-084f6e9c304a)


*Log Browserconsole*
```
Connected: true instrument.ts:132
Translate: time instrument.ts:132
Translate: timeOffset instrument.ts:132
Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).
2
Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Anfrage schlug fehl). Statuscode: (null).

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.
2
Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.

Quellübergreifende (Cross-Origin) Anfrage blockiert: Die Gleiche-Quelle-Regel verbietet das Lesen der externen Ressource auf http://192.168.178.78:8081/. (Grund: CORS-Kopfzeile 'Access-Control-Allow-Origin' fehlt). Statuscode: 200.
```

**Comment by foxriver76** [comment]
ist `192.168.178.78` der Slave?

**Comment by foxriver76** [comment]
@Feuer-sturm 

ist `192.168.178.78` der Slave?

Und Admin nutzt `https`, korrekt?

**Comment by Feuer-sturm** [comment]
@foxriver76 
ist 192.168.178.78 der Slave? -> korrekt
Und Admin nutzt https, korrekt? -> nein, Admin wird ohne https betrieben aber mit "Authentifizierung"

**Comment by foxriver76** [comment]
Danke, ich vermute es braucht leider nochmal eine Änderung im Controller.

**Comment by foxriver76** [comment]
will be fixed in controller v5.0.8
